client/ionic/src/i18n/i18n.tsx

```diff
@@ -0,0 +1,19 @@
+import { createIntl, createIntlCache } from 'react-intl';
+import messages_en from './en.json';
+import messages_zh from './zh.json';
+
+// tslint:disable-next-line:no-any
+export const messages: any = {
+  en: messages_en,
+  zh: messages_zh,
+};
+
+export const language = navigator.language.split(/[-_]/)[0];
```

Comment:
Should we use [Ionic's package](https://ionicframework.com/docs/native/globalization) for extracting the language/locale? It seems to be supported across web and mobile.

Reply:
Actually that's deprecated (https://github.com/apache/cordova-plugin-globalization): 

> With the ECMA Internationalization API now supported on iOS, Android and Windows devices, this plugin is not required any more. Migrating from this plugin to the ECMA Internationalization API is explained in this Cordova blog post.

Reply:
I'm merging this now with the note that we should look into using the Internationalization spec.